packages/aws-cdk-lib/aws-appconfig/lib/private/util.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * The deletion protection check options.
+ */
+export enum DeletionProtectionCheck {
+  /**
+   * The default setting,
+   * which instructs AWS AppConfig to implement the deletion protection value specified in the UpdateAccountSettings API.
```

Comment:
I think the README sentence is better.

```suggestion
   * which uses account-level deletion protection. To configure account-level deletion protection, use the UpdateAccountSettings API.
```